test/EndToEndTests/Tests/Client/Microsoft.OData.Client.E2E.Tests/Common/Server/Default/DefaultDataModel.cs

```diff
@@ -202,7 +203,11 @@ public class Company
     public class PublicCompany : Company
     {
         public string? StockExchange { get; set; }
+
+        [Contained]

```

Comment:
My question about changing the navigation properties to contained applies. Shouldn't we have a different model for containment tests?

Reply:
This is a common CSDL that is shared by several tests:  https://github.com/OData/odata.net/blob/release-7.x/test/EndToEndTests/Services/TestServices/Service%20References/ODataWCFServiceReference/Reference.cs. I had forgotten to update some of these types as contained or not.. For new tests that test anything that is not in these common csdl, I recommend a new CSDL be created.